WordPress/Classes/Services/SharingService.swift

```diff
@@ -0,0 +1,410 @@
+import Foundation
+
+/**
+ SharingService is responsible for wrangling publicize services, publicize 
+ connections, and keyring connections.
+*/
+public class SharingService : LocalCoreDataService
+{
+
+    // MARK: - Methods calling remote services
+
+
+    /**
+    Syncs the list of Publicize services.  The list is expected to very rarely change.
+
+    @param success An optional success block accepting no parameters
+    @param failure An optional failure block accepting an `NSError` parameter
+    */
+    public func syncPublicizeServices(success: (() -> Void)?, failure: (NSError! -> Void)?) {
+        let remote = SharingServiceRemote(api: apiForRequest())
+
+        remote.getPublicizeServices( {(remoteServices:[RemotePublicizeService]) -> Void in
```

Comment:
My new favorite trick: no need to add the `-> Void`.

Props to @astralbodies for the kung fu =)